src/menuFactory.js

```diff
@@ -12,10 +12,13 @@ export default (styles) => {
   return ConfiguredRadium(React.createClass({
 
     propTypes: {
+      additionalToggleMenuActions: React.PropTypes.func,
```

Comment:
There's already a function you can pass to be triggered when the state changes, which this seems to be duplicating. Check out https://github.com/negomi/react-burger-menu#state-change.